db/migrate/20210609081246_add_structured_degree_columns_to_the_course_table.rb

```diff
@@ -0,0 +1,7 @@
+class AddStructuredDegreeColumnsToTheCourseTable < ActiveRecord::Migration[6.1]
+  def change
+    add_column :course, :degree_classification_required, :string
+    add_column :course, :additional_subject_requirements, :boolean
```

Comment:
Not sure I understand this one. From the description is there a radio button that exposes `subject_requirements`? Can that be checked (or not) from the presence (or absence) of a value in `subject_requirements`. That sounds like a view concern if so that the API doesn't need to know about?

Reply:
So it looks like this 
![image](https://user-images.githubusercontent.com/42515961/121327934-cad96480-c90b-11eb-924e-cbe7e8ca971b.png)

Just wondering how you would you select the correct radio button when editing though if you don't persist their  answer for no.

Just default the radio button to no if no value is present for subject_requirements? Or just don't have a default value selected for no when editing?

Or just infer that if you're on the edit page that they must have selected no if there is no value?